dcc-portal-ui/app/scripts/oncogrid/views/oncogrid-analysis.html

```diff
@@ -1,181 +1,221 @@
+      <div id="oncogrid-controls" class="og-grid-align" style="margin-top: 10px; min-height: 70px;">
+        <div style="float:left">
+          <table>
+            <tr>
+              <td>
+                <button id="reload-button" 
+                  data-tooltip="Reload grid" 
+                  class="btn btn-default btn-onco" 
+                  data-ng-click="OncoCtrl.reloadGrid()">
+                  <i class="icon-undo"></i>
+                </button>
+              </td>
+              <td>
+                <button id="cluster-button" 
+                  data-tooltip="Cluster data" 
+                  class="btn btn-default btn-onco" 
+                  data-ng-click="OncoCtrl.clusterData()">
+                  <i class="icon-sort-alt-down"></i>
+                </button>
+              </td>
+
+              <td>
+                <button 
+                  id="heat-button" 
+                  data-tooltip="Toggle heatmap view" 
+                  class="btn btn-default btn-onco" 
+                  data-ng-click="OncoCtrl.heatMap()">
+                  <i class="icon-fire"></i>
+                </button>
+              </td>
+
+              <td>
+                <button id="grid-button" 
+                  data-tooltip="Toggle grid lines" 
+                  class="btn btn-default btn-onco" 
+                  data-ng-click="OncoCtrl.gridLines()">
+                  <i class="icon-grid"></i>
+                </button>
+              </td>
+
+              <td>
+                <button id="crosshair-button" 
+                  data-tooltip="Toggle crosshair" 
+                  class="btn btn-default btn-onco" 
+                  data-ng-click="OncoCtrl.crosshair()">
+                  <i class="icon-target"></i>
+                </button>
+              </td>
+
+              <td>
+                <button id="fullscreen-button" 
+                  data-tooltip="Fullscreen" 
+                  class="btn btn-default btn-onco" 
+                  data-ng-click="OncoCtrl.requestFullScreen()">
+                  <i id="og-fullscreen-button" class="icon-resize-full"></i>
+                </button>
+              </td>
+            </tr>
+
+            <tr style="height: 40px;">
+              <td id="og-crosshair-message" 
+                colspan="6" 
+                style="display:none;">
+                Click and drag to select a region on the OncoGrid to zoom in. 
+              </td>
+            </tr>
+          </table>
         </div>
 
-
-        <div id="oncogrid-div" class="og-pointer-mode"></div>
-        <div id="oncogrid-no-data" class="alert-error" style="display:none;">
-            No result found. <br>
-            Please note that the analysis is filtering on high impact mutations only. <br>
-            Please change your donor or gene set and run the analysis again.
+        <div id="og-legend" class="og-legend">
+          <table id="og-variant-legend" class="onco-table">
+          	<tr>
+              <td>
```

Comment:
I am not quite sure why this is not showing properly indented. In the editor it seems fine.

![image](https://cloud.githubusercontent.com/assets/5777761/17666483/4a09f426-62ce-11e6-8b2e-c317ef34bc72.png)